### DISCUSSION TITLE
Metadata for different languages

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1495] [Q&A] [answered]
Could you please explain how to add translations for files with metadata.

In directory data/meta/ we have number of yml files with metadata in English. If we want to translate them  for Ukrainian version of the site - should we add a folder "ukr" in forder "meta" and put metafiles for indicators there?

Or should we create a file "meta" in data/translations/uk/ directory and just write translation there?

Or maybe is there another way to do it?

**Comment by brockfanning** [comment, marked as answer]
@ukraine-sdg As with most things in the data build there are many ways to do it. :) But I believe that the most intuitive way in your case is to use a subfolder for the language, like you describe. However it needs to use the same language code that you use in other places. So in your case you should use a subfolder called "uk" (not "ukr").
